indico/modules/events/papers/blueprint.py

```diff
@@ -34,11 +34,15 @@
                  templates.RHDownloadPaperTemplate)
 _bp.add_url_rule('/contributions/<int:contrib_id>/paper/submit', 'submit_revision',
                  display.RHSubmitPaper, methods=('GET', 'POST'))
-# Paper reviews
+# Paper reviews/judgments
 _bp.add_url_rule('/papers/<int:contrib_id>/review/type/<review_type>', 'submit_review',
                  display.RHSubmitPaperReview, methods=('GET', 'POST'))
 _bp.add_url_rule('/papers/<int:contrib_id>/revision/<int:revision_id>/review/<int:review_id>/edit', 'edit_review',
                  display.RHEditPaperReview, methods=('GET', 'POST'))
+_bp.add_url_rule('papers/<int:contrib_id>/judge/', 'judge_paper',
```

Comment:
Just pointed out what we did in abstracts - like I said, no strong opinion here.

Personally, I'd probably go for `reset-judgment` since `judge` is an action and so is `reset-judgment` - but `judge/reset` sounds like you are resetting a judge (poor judge! 😀) or casting a judgment to reset something?